rust/test_psibase_macros/services/basic-w-query/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+#[psibase::service(name = "basicwquery")]
+#[allow(non_snake_case)]
+mod service {
+    use psibase::{anyhow, serve_simple_ui, HttpReply, HttpRequest};
+
+    #[action]
+    fn add(a: i32, b: i32) -> i32 {
+        // Wrapper::call() uses and therefore covers the testing of service::Actions
+        assert_eq!(mincallrecvr::Wrapper::call().add2(3, 5), 8);
+
+        a + b
+    }
+
+    #[event(history)]
+    pub fn history_fn1(msg: String) {}
+
+    struct Query;
```

Comment:
This doesn't seem to be used anywhere.

Reply:
Ah, good catch. Thanks. That was a TODO I overlooked before I wrapped up the PR. TODO handled.